calicoctl/commands/node/run.go

```diff
@@ -85,6 +92,29 @@ Options:
                            node resource.  If there is no configured value
                            and the --ip6 option is omitted, the node will not
                            route IPv6.
+     --ip-autodetection-method=<IP_AUTODETECTION_METHOD>
+                           Specify the autodetection method for detecting the
+                           local IPv6 routing address for this node.  The valid
+                           options are:
+                           > first-found
+                             Use the first valid IP address on the first
+                             enumerated interface (common known exceptions are
+                             filtered out, e.g. the docker bridge).  It is not
+                             receommended to use this if you have multiple
+                             external interfaces on your host.
+                           > can-reach=<IP OR DNS>
+                             Use the interface determined by your host routing
+                             tables that will be used to reach the supplied
+                             destination IP or DNS.
+                           > interface=<IFACE NAME REGEX>
```

Comment:
yes you'll almost certainly have to escape some stuff for BASH - but I don't think that should matter here.